modules/vpn/example/example.tfvars

```diff
@@ -0,0 +1,129 @@
+project      = "<project_id>"
+region       = "us-central1"
+network_name = "<network_name>"
+
+vpn_gateway_name = "my-test-gateway"
+router_name      = "my-test-router"
+
+vpn_config = {
```

Comment:
This config is also in `main.tf` - it needs to be aligned somehow - it should be taken from main.tf and moved here solely .

Reply:
Sure. That's an obvious oversight on my side. Fixed